### ISSUE TITLE
on iOS: title bar and status bar overlap

### ISSUE THREAD

**Issue by difiore** [ethoinformatics/ethoinfo-project-app#27]
- title bar at top of diaries and atop breadcrumb on other screens, overlaps with device, wireless symbol, time, etc (screenshot)
  ![img_1349](https://cloud.githubusercontent.com/assets/5214081/15551566/313fe1ea-227c-11e6-9f47-e4708465f3bb.jpg)
  ![img_1355](https://cloud.githubusercontent.com/assets/5214081/15551584/4156c3a0-227c-11e6-929c-88ce7ed03cc7.jpg)

**Comment by tigoe** [comment]
Acknowledged. @jcousins @relevante can fix.

**Comment by jcousins** [comment]
I was able to fix this by editing the iOS app code that gets generated. However, since the app code is generated by Cordova, we need to first update the version of Cordova because this change is related (if I remember correctly) to the iOS7-to-8 transition. I assume the current Cordova version accounts for this. It’s important because among other things, it was probably creating some targeting issues for the breadcrumb, home and hamburger buttons. Closing this in favor of new issue #39.
